src/data-workspace/data-entry-cell/use-data-value-mutation.js

```diff
@@ -78,20 +69,19 @@ export const useDataValueMutation = () => {
 
             // Optimistically update to the new value
             queryClient.setQueryData(dataValueSetQueryKey, () => {
-                const matchIndex =
-                    previousDataValueSet.dataValueSet.dataValues.findIndex(
-                        (dataValue) =>
-                            dataValue.categoryOptionCombo === newDataValue.co &&
-                            dataValue.dataElement === newDataValue.de &&
-                            dataValue.orgUnit === newDataValue.ou &&
-                            dataValue.period === newDataValue.pe
-                    )
+                const matchIndex = previousDataValueSet.dataValues.findIndex(
+                    (dataValue) =>
+                        dataValue.categoryOptionCombo === newDataValue.co &&
+                        dataValue.dataElement === newDataValue.de &&
+                        dataValue.orgUnit === newDataValue.ou &&
+                        dataValue.period === newDataValue.pe
+                )
                 const isNewValue = matchIndex === -1
 
                 // If the field was previously empty the dataValue won't exist yet
                 if (isNewValue) {
                     const formattedNewDataValue = {
-                        attributeOptionCombo: attributeOptionComboId,
+                        attributeOptionCombo,
```

Comment:
Was this a bug?

Reply:
No just a rename, see here: https://github.com/dhis2/data-entry-app/pull/38/files#diff-f4d186e2e137850289b147c9ad12b5e536639e953c7b7e1dce531547b219cf1cL52